libphal/phal_dump.C

```diff
@@ -24,6 +24,7 @@ extern "C" {
 #include <fstream>
 #include <string>
 #include <vector>
+#include <functional>
```

Comment:
we cannot have a commit title as 1, 2, 3 rather it should be what this commit is doing